demos/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
```

Comment:
>  I would like this index.html in root ghpage (no dev mode)

It's already the case. `demo/index.html` is the one deployed (see change in autodeploy). This index has debug mode disable.